src/PhotosphereEditor.tsx

```diff
@@ -1,31 +1,63 @@
 import { useState } from "react";
 
 import AddPhotosphere from "./AddPhotosphere.tsx";
-import { VFE } from "./DataStructures.ts";
+import { Photosphere, VFE } from "./DataStructures.ts";
 import PhotosphereViewer from "./PhotosphereViewer.tsx";
 
+/* -----------------------------------------------------------------------
+    Update the Virtual Field Environment with an added Photosphere.
+
+    * Take the initial VFE from parent
+    * If a change has been made to the initialVFE -> updateTrigger === true
+    * Send the newPhotosphere back to parent 
+    * Parent updates the VFE with the newPhotosphere object
+   ----------------------------------------------------------------------- */
+
+// Properties passed down from parent
 interface PhotosphereEditorProps {
-  vfe: VFE;
+  initialVFE: VFE;
```

Comment:
If we decide to change the name in app maybe we should also make this be named vfe as well?